flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveOptions.java

```diff
@@ -51,4 +51,11 @@
                     .withDescription(
                             "If it is false, using flink native writer to write parquet and orc files; "
                                     + "If it is true, using hadoop mapred record writer to write parquet and orc files.");
+
+    public static final ConfigOption<Integer> TABLE_EXEC_HIVE_PARTITION_SPLIT_THREAD_NUM =
+            key("table.exec.hive.partition-split.thread.num")
```

Comment:
Sorry @Myracle, I have one more question. Is this configuration just like Hive's `hive.load.dynamic.partitions.thread`? I think maybe we can call the configuration `table.exec.hive.load-partition-splits.thread-num` to be closer to the actual behavior, what do you think? 


Besides, could you also add this configuration to the Hive documentation ?

Reply:
Good suggestion! I have modified it. Can you review it again? Thank you.